build/hygiene.js

```diff
@@ -267,6 +288,12 @@ function hygiene(some, linting = true, secrets = true) {
 					process.stdout.write('.');
 				}
 				this.emit('data', data);
+
+				// --- Start Positron ---
+				if (errorCount > 0) {
+					this.emit('error', `Hygiene failed with ${errorCount} errors`.red);
+				}
+				// --- End Positron ---
```

Comment:
This ensures we emit an error and fail the precommit hook if we find any errors.